VideoLocker/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -104,8 +104,9 @@ public boolean isEnabled() {
     /**
      * Course Enrollment configuration.
      */
-    public class EnrollmentConfig {
+    public class CourseDiscoveryConfig {
```

Comment:
Mmm... I changed this back to EnrollmentConfig, and changed NATIVE to NATIVE_COURSE_DISCOVERY_ENABLED